docs/user/grading.rst

```diff
@@ -0,0 +1,35 @@
+Grading
+=======
+
+Artemis allows instructors to define grading keys for courses and exams.
+When the grading key is defined, the total points obtained by a student from all exercises over maximum points are calculated and mapped to a grade.
+Grading Keys can be either:
+
+- ``Grade`` type which maps the points to a letter or numeric grade with a ``First Passing Grade``, useful for final exams or courses without exams that are graded solely by exercises.
+- ``Bonus`` type which maps the points to a numeric value, useful when grade obtained in the given course or exam is not an end result but complements a final exam.
+
+Instructors
+-----------
+Instructors can export and import grading keys in CSV format to reuse them inside or outside of Artemis.
+If a grading key is defined, exporting student results includes the obtained grade information as well.
+It is also possible to create or modify grading keys after an exam or course is over, student grades will be adjusted automatically.
+
+    .. note::
+        You can check `how to create grading keys for courses and exams <user/exams/instructors_guide/#grading-key>`__ for more detailed information.
```

Comment:
Would it make sense to replace the section in the exam documentation by a link to this page and instead move all information related to the grading keys in here? That would also deduplicate some of the information.

Reply:
I asked @krusche about that and he suggested to keep that page as it is and add the links here.